src/libostree/ostree-sysroot-deploy.c

```diff
@@ -2079,7 +2079,10 @@ boot_find_path_on_disk(char** path_out, GError **error)
   *path_out = g_steal_pointer(&path);
   return TRUE;
 #else /* !HAVE_LIBMOUNT */
-  return g_strdup("");
+  g_printerr ("warning: ostree compiled without libmount so we can't determine "
```

Comment:
Hm.  It feels like we should make libmount a hard dependency *but* also add a `--disable-ostree-as-host` build option or something so that the people who just want libostree for flatpak can disable it?

Or alternatively just make it a hard dependency across the board, I can't really think of a case where someone would have an issue shipping libmount.